cli/signal.rs

```diff
@@ -11,7 +21,33 @@ pub fn kill(pid: i32, signo: i32) -> Result<(), OpError> {
 }
 
 #[cfg(not(unix))]
-pub fn kill(_pid: i32, _signal: i32) -> Result<(), OpError> {
-  // TODO: implement this for windows
+pub fn kill(pid: i32, signal: i32) -> Result<(), OpError> {
+  unsafe {
+    match signal {
+      9 | 15 | 2 => {
```

Comment:
Can you use symbolic names here (e.g. `SIGTERM`). Or at least add a comment describing what these signals mean?
Note that currently Deno.signals contains linux signals (that's a bug).